AICP: {
  Registration: {
    type: "Pointmatcher", # GICP, Kaess
    sensorRange: 30.0,         # in meters
    sensorAngularView: 270.0,  # in degrees

    Pointmatcher: {
      configFileName: "icp_autotuned_default.yaml",
    }
  },
  Overlap: {
    type: "OctreeBased",

    OctreeBased: {
      octomapResolution: 0.2,
    }
  },
  Classifier: {
    type: "SVM",

    SVM: {
      threshold: 0.50,
      trainingFile: "${DRS_BASE}/code/src/aicp/data/labels/training_all_labelled_1Sep.txt",
      testingFile: "${DRS_BASE}/code/src/aicp/data/labels/testing_labelled_27Aug.txt",
      saveFile: "${DRS_BASE}/code/src/aicp/data/classification/svm_1000training_thresh50_cross_validation_opencv3.xml",
      saveProbs: "${DRS_BASE}/code/src/aicp/data/classification/probs_test.txt",
      modelLocation: "${DRS_BASE}/code/src/aicp/data/models/svm_opencv3.xml",
    },
  },
}
